task_list_app/task_list_app/src/main/java/com/jackson/task_list_app/api/controller/TaskController.java

```diff
@@ -0,0 +1,30 @@
+package com.jackson.task_list_app.api.controller;
+
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.jackson.task_list_app.api.models.MyTask;
+import com.jackson.task_list_app.api.services.TaskService;
+
+
+
+@RestController
+public class TaskController {
+
+    @Autowired
+    private TaskService taskService;
+
+    @CrossOrigin
```

Comment:
Please try the backend without this annotation. We added it back when we were testing. It might not be needed.